virt/virt-2-4-release-notes.adoc

```diff
@@ -1,11 +1,206 @@
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1796410[*BZ#1796410*])
+
+** As a workaround, trigger the upgrade process to {VirtProductName} {VirtVersion}
+from the CLI by running the following `oc` patch command:
++
+----
+$ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.4 && oc patch -n "${TARGET_NAMESPACE}" $(oc get subscription -n ${TARGET_NAMESPACE} --no-headers -o name) --type='json' -p='[{"op": "replace", "path": "/spec/channel", "value":"'${CNV_CHANNEL}'"}, {"op": "replace", "path": "/spec/installPlanApproval", "value":"Automatic"}]'
+----
++
+This command points your subscription to upgrade channel `2.4` and enables automatic updates.
+
+//This bug - 1686208- was listed in POST status in Bugzilla:
+* After migration, a virtual machine is assigned a new IP address. However, the
+commands `oc get vmi` and `oc describe vmi` still generate output containing the
+obsolete IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1686208[*BZ#1686208*])
++
+** As a workaround, view the correct IP address by running the following command:
++
+----
+$ oc get pod -o wide
+----
+
+//This defect was closed therefor the content for this defect was deleted:(link:https://bugzilla.redhat.com/show_bug.cgi?id=1743985[*BZ#1743985*])
+//This defect is in POST status [*BZ#1760028*]
+* Live migration fails when nodes have different CPU models. Even in cases where
+nodes have the same physical CPU model, differences introduced by microcode
+updates have the same effect. This is because the default settings trigger
+host CPU passthrough behavior, which is incompatible with live migration.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1760028[*BZ#1760028*])
++
+** As a workaround, set the default CPU model in the `kubevirt-config` ConfigMap,
+as shown in the following example:
++
+[NOTE]
+====
+You must make this change before starting the virtual machines that support
+live migration.
+====
++
+. Open the `kubevirt-config` ConfigMap for editing by running the following command:
++
+----
+$ oc edit configmap kubevirt-config -n openshift-cnv
+----
++
+. Edit the ConfigMap:
++
+[source,yaml]
+----
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+data:
+  default-cpu-model: "<cpu-model>" <1>
+----
+<1> Replace `<cpu-model>` with the actual CPU model value. You can determine this
+value by running `oc describe node <node>` for all nodes and looking at the
+`cpu-model-<name>` labels. Select the CPU model that is present on all of your
+nodes.
+
+//Jira story for the following content https://issues.redhat.com/browse/CNV-4757. Nelli noted that for CNV 2.5, this can be removed from Known Issues. The defect status, however, is CLOSED.
+* {VirtProductName} cannot reliably identify node drains that are triggered by
+running either `oc adm drain` or `kubectl drain`. Do not run these commands on
+the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
+drain if there are virtual machines running on top of them.
++
```

Comment:
I would avoid this additional empty line

Reply:
It's keeping the next para indented properly.
Perhaps we should use a sub-bullet like with the other workaround text?